api/testdata/AddGroupRBAC/cluster-scope/base/core/namespaces/testproject/kustomization.yaml

```diff
@@ -4,6 +4,6 @@ namespace: testproject
 resources:
 - namespace.yaml
 components:
-- ../../../../components/project-admin-rolebindings/testgroup
 - ../../../../components/limitranges/default
```

Comment:
Can you extract out refactors to files unrelated to the `onboard` command as a separate commit, the pr is already pretty big, I think these changes are adding unnecessary bloat.

Reply:
yes I can, taking a look.